ada_2_fhir-r4/zibs2020/fhir_instance/nl-core-HelpFromOthers-02.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CarePlan xmlns="http://hl7.org/fhir">
+   <id value="nl-core-HelpFromOthers-02"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-HelpFromOthers"/>
+   </meta>
+   <status value="active"/>
+   <intent value="plan"/>
+   <category>
+      <coding>
+         <system value="http://snomed.info/sct"/>
+         <code value="243114000"/>
+         <display value="ondersteuning"/>
+      </coding>
+   </category>
```

Comment:
Error from the QA tool:

```
 -  error at CarePlan.category[0] (9, 14):
     Slicing cannot be evaluated: Could not match discriminator ([coding]) for slice CarePlan.category:helpFromOthers in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-HelpFromOthers - the discriminator [coding] does not have fixed value, binding or existence assertions
```

Reply:
Error has been fixed, see https://github.com/Nictiz/Nictiz-R4-zib2020/pull/308.